src/Data/ProgrammesDb/Entity/RefOptions.php

```diff
@@ -8,9 +8,13 @@
 
 /**
  * @ORM\Entity()
+ * @ORM\Table(indexes={@ORM\Index(name="entity_id_idx", columns={"entity_id"})})
```

Comment:
`ref_options_entity_id_idx` as these index names are global across the DB and it'd be useful to see at-a-glance what table an index refers to